src/store/modules/user/actions.ts

```diff
@@ -231,11 +231,7 @@ const actions: ActionTree<UserState, RootState> = {
    * update current shopify config id
    */
   async setCurrentShopifyConfig({ commit, dispatch, state }, payload) {
-    let shopifyConfig = payload.shopifyConfig;
-    if(!shopifyConfig) {
-      shopifyConfig = state.shopifyConfigs.find((configs: any) => configs.shopifyConfigId === payload.shopifyConfigId)
-    }
-
+    const shopifyConfig = state.shopifyConfigs.find((configs: any) => configs.shopifyConfigId === payload.shopifyConfigId);
```

Comment:
Revert this change and just update the variable to `let shopifyConfig = payload.shopifyConfigId;`